src/components/filters/RangeFilter.vue

```diff
@@ -1,27 +1,47 @@
 
 <template>
   <div>
-    <b-input-group>
+    <b-input-group class="mb-1">
```

Comment:
only for range filter ?

Reply:
Only for the first one. A but of space below it makes it look better.